src/DataFrames.jl

```diff
@@ -75,7 +75,61 @@ export @~,
        stackdf,
        unique!,
        unstack,
-       writetable
+       writetable,
+       # FIXME: unexport, these should go in Base or nowhere
```

Comment:
Has there been a larger discussion on this? DataFrames seems like a natural place for these....